pkg/kubelet/cm/dra/manager.go

```diff
@@ -67,8 +67,9 @@ type ManagerImpl struct {
 }
 
 // NewManagerImpl creates a new manager.
-func NewManagerImpl(kubeClient clientset.Interface, stateFileDirectory string, nodeName types.NodeName) (*ManagerImpl, error) {
-	klog.V(2).InfoS("Creating DRA manager")
+func NewManagerImpl(ctx context.Context, kubeClient clientset.Interface, stateFileDirectory string, nodeName types.NodeName) (*ManagerImpl, error) {
+	logger := klog.FromContext(ctx)
+	logger.V(2).Info("Creating DRA manager")
```

Comment:
This seems redundant. The caller already logs "Creating Dynamic Resource Allocation (DRA) manager".

Logging in the caller seems like the better place, because that is where the decision to create one is made. A function that always gets called shouldn't log that it gets called because that doesn't really add information.

When removing this log call, the `ctx` parameter isn't necessary.